dockerfiles/mlrun-api/Dockerfile

```diff
@@ -84,10 +84,4 @@ VOLUME /mlrun/db
 # and avoid zombie processes
 ENTRYPOINT ["tini", "--"]
 
-CMD [ \
-    "uvicorn", \
-    "server.api.main:app", \
-    "--proxy-headers", \
-    "--host", "0.0.0.0", \
-    "--log-config", "server/api/uvicorn_log_config.yaml" \
-]
+CMD ["bash", "./dockerfiles/mlrun-api/start_api.sh"]
```

Comment:
since you have moved from running uvicorn under tini to bash, you may now not use tini at all and remove it altogether (see line 40).

since you are using `exec` it automatically replaces the bash with your program meaning the signals would automatically be forwarded to the process (as it replaces the bash with the program you `exec`ing on).

that being said, I would suggest is comment on bash script that `exec` is used to replace the bash as program so that events would be delegated correctly.

once you have all in, make sure that delete/delete --force works as expected and it doesnt take too much time before it goes down (==signals are effectively handled)

Reply:
scrape everything out. I think that leaving in tini and bash with exec would make perfect sense.

just leave a comment on the bash script to ensure the `exec` remains there for zombie process reaping purpose :)